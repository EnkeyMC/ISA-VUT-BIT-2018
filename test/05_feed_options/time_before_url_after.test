NAME=Show time and URL (-T before, -u after)
ARGS=-T http://www.mocky.io/v2/5bd3096a3400003600cfdff9 -u
RC=0
CHECK_OUTPUT=1
