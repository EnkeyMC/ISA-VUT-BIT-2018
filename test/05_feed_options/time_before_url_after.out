*** Scripting News ***
Law and Order
URL: http://scriptingnews.userland.com/backissues/2002/09/29#lawAndOrder
Aktualizace: Sun, 29 Sep 2002 23:48:33 GMT

Rule 1
URL: http://scriptingnews.userland.com/backissues/2002/09/29#rule1
Aktualizace: Sun, 29 Sep 2002 17:24:20 GMT

Really early morning no-coffee notes
URL: http://scriptingnews.userland.com/backissues/2002/09/29#reallyEarlyMorningNocoffeeNotes
Aktualizace: Sun, 29 Sep 2002 11:13:10 GMT